public/app/frontend/src/components/navigation-sections/navigation-sections.scss

```diff
@@ -0,0 +1,34 @@
+@use 'sass:map';
+@import '../../../style';
+
+.navigation-sections {
+    display: block;
+    max-width: 100%;
+
+    &__list {
```

Comment:
When I preview this in storybook, on small screens there's no spacing at the sides. Just double checking, is this intentional?

Reply:
Thanks for reviewing @EarthlingDavey :) 

Yes, it's because it stretches to fit the width of it's container so any spacing will come from the page container on mobile or grid gaps on desktop.

Reply:
No probs... thanks for explaining